### DISCUSSION TITLE
[@types/gapi] gapi/index.d.ts is not a module.ts(2306)

### DISCUSSION THREAD

**Discussion by ajaykumarparashar11** [DefinitelyTyped/DefinitelyTyped#62993] [Issues with a @types package]
Hello

i have installed a type by using **"npm install --save @types/gapi"** for type definitions for Google API Client using this package **https://www.npmjs.com/package/@types/gapi**

while importing this type definitions in my angular project file(module) **I am getting this error** 

**File '...path.../node_modules/@types/gapi/index.d.ts' is not a module.ts(2306)**

![Screenshot 2022-10-31 at 9 04 44 PM](https://user-images.githubusercontent.com/23501197/199047260-6a17599a-a1c1-4407-9fa5-58ae50f1584e.png)

can anybody tell me what wrong i am doing in this?

Angular version: 13.1.0

any help will be much appreciated, thanks in advance.

[a comment by an automated account is not shown]

**Comment by MysteryBlokHed** [comment]
The `@types/gapi` package only declares types, but doesn't export any. Try to see if you can use the `gapi` object without importing anything.

**Reply by Semigradsky** [reply]
In README:

![image](https://user-images.githubusercontent.com/1198848/199818470-ff211263-ed8d-4737-8a1c-96085dd92546.png)

**Reply by MysteryBlokHed** [reply]
Looking at the DefinitelyTyped header, it seems like `@types/gapi` is not actually related to the `gapi` module on npm:

<https://github.com/DefinitelyTyped/DefinitelyTyped/blob/6e83bde792b326b64796dc67420c8521497e8852/types/gapi/index.d.ts#L1-L5>

`@types/gapi` might have to be renamed to avoid this conflict.
